phobos/simple_deployment.md

```diff
@@ -0,0 +1,182 @@
+# Install rpc-ceph in the Phobos lab OnMetal 
+
+
+## Create Environment
+* 1 admin vm
+* 1 rsyslog vm
+* 5 boxes for storage nodes
+
+```bash
+openstack server create --flavor  vm-osa-aio --image a4a9bc69-de27-4cec-903a-cf57be693738 \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-admin01-c_
+
+openstack server create --flavor  vm-osa-aio --image a4a9bc69-de27-4cec-903a-cf57be693738 \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey rsyslog01-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor01-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor02-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor03-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor04-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor05-c_
+```
+
+
+## Storage Nodes favor ironic-rpc-storage-interim additional drives
+/dev/sdb | 3TB
+...
+/dev/sdk | 3TB
+
+
+## Phobos Xenial image is set to not allow root user to log in
+To remove the restriction run this on each osd node and on rsyslog
+```bash
+ssh ubuntu@<ip>  "sudo sed -i -e 's/^no-port-forwardin.*\(ssh-rsa.*$\)/\1/g' /root/.ssh/authorized_keys"
+```
+
+
+##  hosts file
+Create entries in /etc/hosts for all servers, specifying the IP address to which we can reach the servers via ssh:
+Example **/etc/hosts** snippet:
+```bash
+/etc/hosts
+172.20.41.X    ceph-admin01-c_ 
+172.20.41.X    rsyslog01-c_
+172.20.41.X    ceph-stor01-c_
+172.20.41.X    ceph-stor02-c_
+172.20.41.X    ceph-stor03-c_
+172.20.41.X    ceph-stor04-c_
+172.20.41.X    ceph-stor05-c_
```

Comment:
This is fine, but as a side note - perhaps we could automate this.

Reply:
I would certainly like to do that but not at this time.

Reply:
:+1: